gdbgui/static/js/gdbgui.js

```diff
@@ -2350,6 +2350,7 @@ const Expressions = {
             let children = r.payload.children.map(child_obj => Expressions.prepare_gdb_obj_for_storage(child_obj, expr_type))
             // save these children as a field to their parent
             parent_obj.children = children
+            parent_obj.numchild = children.length
```

Comment:
`-var-list-children`'s response is processed in :

    if('has_more' in r.payload && 'numchild' in r.payload && 'children' in r.payload){
        Expressions.gdb_created_children_variables(r)
    }

On absent of this line :

`parent_obj.children` becomes non-empty array while `parent_obj.numchild` still be 0, 
so children aren't displayed. 

Will this line introduce any troubles ?